src/yb/yql/pggate/util/ybc_stat.cpp

```diff
@@ -107,7 +107,7 @@ ybcstat_get_wait_event_type(uint32_t wait_event_info) {
             event_type = "Flush and Compaction";
             break;
         case YB_CQL_WAIT_STATE:
-            event_type = "Query Processing";
+            event_type = "Query";
```

Comment:
Query seems too generic. Could as well be confused to a pg-query? 
Probably have CQL somewhere in the name?